apm-sniffer/apm-sdk-plugin/postgresql-8.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/postgresql/define/PgStatementInstrumentation.java

```diff
@@ -43,12 +43,8 @@ public InstanceMethodsInterceptPoint[] getInstanceMethodsInterceptPoints() {
             new InstanceMethodsInterceptPoint() {
                 @Override
                 public ElementMatcher<MethodDescription> getMethodsMatcher() {
-                    return named("execute").and(takesArgumentWithType(0, "java.lang.String"))
-                                           .or(named("execute").and(takesArgumentWithType(0, "java.lang.String[]")))
```

Comment:
Is this removed on purpose?

Reply:
I noticed in `org.postgresql.jdbc.PgStatement`, there is no `String[]` at index 0 for `execute` and `executeUpdate` method.

Reply:
Meanwhile `public int executeUpdate(String sql, String[] columnNames)` seems having a different execution path. Is this another bug? (should be `takesArgumentWithType(1...)?`

Reply:
OK, There are some problems of my implementation here.  
I have tightened the matching rule for execute and executeUpdate; delete the executeLargeUpdate methods which have not been implemented in PgStatement.

Please check, thank you